job-simulation.rb

```diff
@@ -1,4 +1,24 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative './Stack.rb'
+require_relative './Queue.rb'
 
+waiting_workers = Queue.new
+workers = Stack.new
 
+# scales for however many workers are on the list
+10.times do |worker|
+	waiting_workers.enqueue(worker + 1)
+end
+
+6.times do 
+	workers.push(waiting_workers.dequeue)
+end 
+
+evil_boss_roulette = 1 + rand(6)
```

Comment:
Nice! You're not even nice and let them get away with rolling a 0 and all being saved (You did it the way I intended, but not everyone will notice this nuance, I suspect.